### DISCUSSION TITLE
How to use voice notes?

### DISCUSSION THREAD

**Discussion by mikedufty** [opengisch/QField#4132] [Q&A]
The new voice recording feature sounds good, however I can't find any instruction on how to use it.  Does it need an attachment field?

**Comment by lucienicolier** [comment]
Hi @mikedufty ,

To use voice recording, you will need to configure your form on QGIS 3.30.
![image](https://user-images.githubusercontent.com/50236352/231474119-b0dc25ea-f34a-42ec-9c59-bed6baf4dbab.png)

**Reply by PackElend** [reply]
is there any news?
The only thing I find is https://docs.qfield.org/how-to/project-setup/pictures/ and bug report here: https://gis.stackexchange.com/questions/491995/qfield-audio-recording-does-not-save-file-to-project, plus the announcement of this featuer https://www.opengis.ch/de/2023/04/04/capturing-more-while-in-the-field-with-the-new-qfield-2-7/

**Reply by nirvn** [reply]
@PackElend , FYI, we've moved this kind of discussion over here: https://community.qfield.org/ -- you're more likely to get an answer from the broader community.

That being said, audio recording should be working with latest versions of QField. One easy way to try it is to use QField's project creation functionality (enable note and check the [x] take attachments). Then, you can add a note on the newly created project within which you can try to record and attach an audio clip (using the 3-dot menu icon next to the camera icon).

<img width="713" height="1600" alt="image" src="https://github.com/user-attachments/assets/5ec6f4df-56ef-45a5-9ab0-bb7740e0eaac" />

**Reply by PackElend** [reply]
Thx for the quick response 🙂, didn't expect that

What do you mean by

> One easy way to try it is to use QField's project creation functionality (enable note and check the [x] take attachments). 

Im very new to QField
We can continue the discussiona here https://community.qfield.org/t/automated-transcribe-voice-notes-feasible/1584

**Comment by PackElend** [comment]
Can it be used to transcribe the recording using an external service and import the result?
